serverside_challenge_2/challenge/lib/utils/energy_price.rb

```diff
@@ -0,0 +1,75 @@
+require 'csv'
+require 'nkf'
+
+module Utils
+  class EnergyPrice
+    TYPE_BASIC = 'basic'
+    TYPE_CONSUMPTION = 'consumption_fee'
```

Comment:
慣習的なので、知識としてただ知っておけばいいことですが、定数はfreezeしておきましょう。
```suggestion
    TYPE_CONSUMPTION = 'consumption_fee'.freeze
```